src/request/import/ImportApi.js

```diff
@@ -14,8 +14,8 @@ class ImportApi extends TopLevelApi {
 
         const parsedRequest = {};
         parsedRequest.appName = this.parseAppName(request.appName);
-        parsedRequest.defaultKeyPath = this.parsePath(request.defaultKeyPath, 'defaultKeyPath');
```

Comment:
This one we just don't use I guess?

Reply:
Exactly. It was used back when we had the wallet type selector (selection between legacy and bip39 in Keyguard).

Reply:
ok :)